crates/egui/src/viewport.rs

```diff
@@ -965,6 +965,15 @@ pub enum ViewportCommand {
     ///
     /// The results are returned in `crate::Event::Screenshot`.
     Screenshot,
+
+    /// Request a cut from the clipboard
+    RequestCut,
+
+    /// Request a copy from the clipboard
+    RequestCopy,
+
+    /// Request a paste from the clipboard
+    RequestPaste,
```

Comment:
These needs better docstrings, because I don't get it. Are these effectively injecting events in the next frame?

Reply:
They are doing the equivalent of pressing `CTRL + C`, `CTRL + X` and `CTRL + V` (or the CMD equivalent on macOS). The reason we had to introduce this is because adding a second clipboard instance broke clipboard interaction for Wayland users. So in order to offer users a context menu with these actions, we needed a way to trigger them. Since we cannot programmatically emit keypresses currently, this seemed like a viable solution.

So maybe the docstring could say `Trigger a CMD + X cut` or something like that

Reply:
I see.

Perhaps "Reads text from the system clipboard and pastes it in the next frame"